buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1425,6 +1425,7 @@ artifactId=${project.name}
     /** ***********************************************************************************************/
 
     // Method to create the PortableValidatesRunnerTask.
+    // Generated task expect :beam-sdks-java-core and :beam-runners-core-java to be in the class path.
```

Comment:
Reword this to make it clear this is a gradle evaluation dependency (rather than a java classpath issue).

Specifically, users must make sure the following is in the consuming module's `build.gradle` file:

```
evaluationDependsOn(":beam-sdks-java-core")
evaluationDependsOn(":beam-runners-core-java")
```